### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by madhavmansuriya40**

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts [hunk 1]
```diff
@@ -510,23 +510,14 @@ export class AddEditExpensePage implements OnInit {
...
 
       const splitExpensePopover = await this.popoverController.create({
         component: SplitExpensePopoverComponent,
         componentProps: {
-          class: popupTypeItemClass,
           areProjectsAvailable,
           areCostCentersAvailable
         },
-        cssClass: 'split-expense-popover'
+        cssClass: 'dialog-popover'
```
Comment (iamtarun1993) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.ts:520]: why the cssClass name got changed...
IIRC I put extra required scss for this class in global.scss file

File: src/app/fyle/add-edit-expense/split-expense-popover/split-expense-popover.component.html [hunk 2]
```diff
@@ -1,25 +1,27 @@
-<div class="split-expense-popover split-expense-popover--{{class}}">
```
Comment (iamtarun1993) [inline src/app/fyle/add-edit-expense/split-expense-popover/split-expense-popover.component.html:1]: split-expense-popover--{{class}} is required to make the popover height based on no of element available ..

why this got removed?

Comment (madhavmansuriya40) [reply src/app/fyle/add-edit-expense/split-expense-popover/split-expense-popover.component.html:1]: I managed to increase decrease height dynamically in CSS itself, this was written very badly in the old app as well

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts (hunk 1 above)
Comment (madhavmansuriya40) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:520]: no sir, we don't need extra CSS I managed it in CSS and that fix went in master long ago

Comment (iamtarun1993) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:520]: oh then its ok.

**Review by iamtarun1993 — APPROVED** [review]

**Pull request closed without merging**